server/routes/index.ts

```diff
@@ -30,12 +30,115 @@ export default function routes(): Router {
       completedHours: 0,
       totalHours: 100,
     }
-    res.render('pages/pop/progress', { progress })
+
+    // const appointment = ""; // if no appointment
+
+    const appointment = {
+      title: 'Community Garden Maintenance',
+      date: 'Friday 15 March 2024',
+      time: '09:00',
+      location: '123 Garden Street, London SE1 7TH',
+    }
+    res.render('pages/pop/progress', { progress, appointment })
   })
 
   get('/conditions', async (req, res, next) => {
     res.render('pages/pop/conditions')
   })
 
+  get('#', async (req, res, next) => {
```

Comment:
This needs to come out for this view appointments feature - move to another future branch maybe?